wca-guidelines.md

```diff
@@ -32,6 +32,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-1><officials><officials> Article 1: Officials
 
+- 1c+) [CLARIFICATION] "The WCA Delegate" refers to anyone fulfilling the role of the WCA Delegate at a competition.
```

Comment:
Maybe remove the last "the", so that it's "fulfilling the role of WCA Delegate"?
To me "the WCA Delegate" still sounded like there is only one Delegate at a competition :p

Reply:
Good point, it's changed now.